Help/test/new.js

```diff
@@ -39,7 +39,7 @@ describe ("Help New Suite", function() {
     // Call the test
     callbacks.pendingLogin();
 
-    expect(ls.getItem("Login:redirect_to_app")).toBe(window.location.href);
+    expect(Privly.storage.get("Login:redirect_to_app")).toBe(window.location.href);
```

Comment:
The difference in conventions between "Login:redirect_to_app" and "Login/redirect_to_app" bothers me. I think we need to adopt one or the other. I am OK with moving to the "/" convention.